ReactApp/src/index.css

```diff
@@ -15,23 +15,34 @@
 
 body {
 	margin: 0;
-	padding: 16px;
-	padding-top: 8px;
-	padding-bottom: 46.5px;
 	font-family: -apple-system;
 	line-height: 1.55;
 }
 
 /* Theme (for no-theme design) */
+* {
+	-webkit-tap-highlight-color: transparent;
+}
+
+.components-button:focus:not(:disabled) {
+	outline: 0 !important;
```

Comment:
At some point in the future, we should take time to consider how we might best coexist with core's styles. I.e., the use of `!important` concerns me, it is likely worthy of exploring a more sustainable override approach. 

Not a task for this PR, merely sharing the thought. Related to https://github.com/Automattic/dotcom-forge/issues/8678.

Reply:
Definitely! I think once we have a working prototype with the needed features we should spend some time on that. Take some time to clean up and improve what we have now.